### PR TITLE
Awesome books es6 v1

### PR DESCRIPTION
**Pull request by SulimanJoya**
## Changes
- Created a new repo (
Awesome-books-with-ES6)
- Add linter for HMTL, CSS, and JavaScript 
- Add different sections in the index file 
- Add CSS file 
- Add style for different sections  
- Add script index.js file
- Add module folder
- Implement Awesome books app with ES6
- Add date functionality by using Luxon library
- Add the new version of the README file 
- Fixed the linter errors 

## Checklist before requesting a review
- [X] Liner errors 
- [X] GitHub flow
- [X] Documented professionally 
- [X] Followed HTML & CSS best practices 
- [X] Followed JavaScript best practices

### CODE REVIEW

**Review by rustam-tolipov — COMMENTED** [review]
Hi @SulimanJoya, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that needed to include a future features section in the readme.

Please note that all projects must include a Future Features section outlining 1-3 potential features to be added. You have submitted your project without this section, so please add it before submitting again. Thank you for your understanding.




Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
 <a name="readme-top"></a>
 
+<div align="center">
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+</div>
+
```
Comment (OLIPLICHE) [inline README.md:7]: # Optional
- I strongly advise you to remove the logo(Microverse). You can replace it with your own logo if you have it. This is part of the general requirement.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+};
+
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  addBook();
+  bookUpload();
+  pushToLocal();
+});
+
+const now = DateTime.now();
```
Comment (OLIPLICHE) [inline index.js:33]: - Okay 👍 
Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi @SulimanJoya, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# Highlights
- The App is more structured with ES6
- Linter is green (the last commit have a green check mark (✅)) 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SulimanJoya** [comment]
I have already did this project before a I take leave.

**Review by Qoosim — COMMENTED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @SulimanJoya, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**